clusterloader2/testing/huge-service/modules/service.yaml

```diff
@@ -0,0 +1,84 @@
+{{$endpoints := .endpoints}}
+{{$isHeadless := .isHeadless}}
+{{$namespaces := .namespaces}}
+{{$serviceName := .serviceName}}
+
+steps:
+- name: Create huge services
```

Comment:
Please make "huge services" a variable.

This way you will be able to reuse this module:
- name "huge servieces"
   headless: false
- name: "headless servvice"
  headless: true

Reply:
Going to reuse the `$serviceName` var. Fixed.